indico/modules/users/controllers.py

```diff
@@ -517,3 +520,17 @@ def _process(self):
         send_email(make_email(self.request.email, template=tpl))
         flash(_('The request has been rejected.'), 'success')
         return jsonify_data()
+
+
+class RHUserSearch(RHProtected):
+    """Search for users based on given criteria"""
+
+    @use_args({
+        'email': fields.Str(validate=lambda s: len(s) > 3 and '@' in s),
+        'name': fields.Str(validate=validate.Length(min=3))
+    })
+    def _process(self, args):
+        if not args:
```

Comment:
i guess we can't do that on the webargs level?

Reply:
https://github.com/sloria/webargs/issues/113